app/src/main/java/io/horizontalsystems/bitcoinkit/demo/MainViewModel.kt

```diff
@@ -44,10 +44,11 @@ class MainViewModel : ViewModel(), BitcoinKit.Listener {
 
     private lateinit var bitcoinKit: BitcoinKit
 
-    private val walletId = "FastSyncWallet13"
-    private val networkType = BitcoinKit.NetworkType.MainNet
+
+    private val walletId = "SyncDemoWallet1"
+    private val networkType = BitcoinKit.NetworkType.TestNet
```

Comment:
It's ok that this is just TestNet or MainNet? It doesn't need to be dynamic based on if we are on prod or dev?

Reply:
This file doesn't matter. It is used to run the demo app which is used for testing purposes.